core_backend/app/contents/schemas.py

```diff
@@ -15,12 +15,22 @@ class ContentCreate(BaseModel):
     content_tags: list = []
     content_metadata: dict = {}
 
-    model_config = ConfigDict(from_attributes=True)
+    model_config = ConfigDict(
+        from_attributes=True,
+        json_schema_extra={
+            "examples": [
+                {
+                    "content_title": "An example content title",
+                    "content_text": "And an example content text!",
+                },
```

Comment:
for some reason the order of these are reversed on the UI which make it a little confusing... any fix there?

Reply:
Yeah I noticed that, but I'm not sure :(

Reply:
I think this is kinda crucial for easy comprehension... I would actually prefer to not have examples but have them the right way round